### PR TITLE
Updated strings for Cobalt

### PR DESCRIPTION
**Pull request by boek**

### CODE REVIEW

**Comment by flodolo** [comment]
Sorry, it looks like we're starting all over from scratch, and that's not OK (and clearly not your fault).

Can you please talk with @bkmunar on the process he used last time? Was it documented anywhere? If not, why, given how much we struggled for the past two cycles?

From what I wrote on Slack a week ago:

```
1. clone this repository at the same level as Focus https://github.com/mozilla-mobile/firefox-ios-build-tools. You'll have the focus and firefox-ios-build-tools in the same folder
2. from within the focus folder run ../firefox-ios-build-tools/scripts/export-locales-focus.sh
3. it should clone the l10n repository into focusios-l10n, already branched, 1 commit per locale, fixed templates (you need virtualenv installed though, not sure if there are other dependencies)
4. that you can add your fork as remote, and push it
```

Looks at how the PR should look (number of commits)
https://github.com/mozilla-l10n/focusios-l10n/pull/15

File: en-US/focus-ios.xliff [hunk 1]
```diff
@@ -1,454 +1,424 @@
...
-        <target>Search for %@</target>
         <note>Label displayed for search button when typing in the URL bar</note>
       </trans-unit>
       <trans-unit id="URL.trackersBlockedLabel">
         <source>Trackers blocked</source>
-        <target>Trackers blocked</target>
         <note>Text for the URL bar showing the number of trackers blocked on a webpage.</note>
       </trans-unit>
+      <trans-unit id="actionSheet.openInChrome">
+        <source>Open in Chrome</source>
```
Comment (flodolo) [inline en-US/focus-ios.xliff]: Why these 3 strings are not `Open in %@`?

**Comment by boek** [comment]
Hey @flodolo,

My apologies for starting from scratch. I followed the steps layed out in the Focus wiki and I think they’re a little out of date. I will update the wiki and get you an updated PR here soon.

My short-mid term goal is to automate as much of the manual process as possible to help eliminate these kinds of mistakes in the future. That way we can focus on the content and not the process.

**Comment by flodolo** [comment]
Note: this PR needs to be closed and replaced by a proper one, but the question about those 3 strings remain and should be answered before that happens.

And we *need* the process to be documented on the mobile's team side, the l10n team is responsible for getting the content localized and managing relation with communities, it doesn't own the technical bits to get there (even if I ended up writing a lot of it).

File: en-US/focus-ios.xliff (hunk 1 above)
Comment (boek) [inline en-US/focus-ios.xliff]: Great catch. Updated PR will have this fixed.

**Comment by flodolo** [comment]
> My apologies for starting from scratch. I followed the steps layed out in the Focus wiki and I think they’re a little out of date. I will update the wiki and get you an updated PR here soon.

No problem, as said it's hardly your fault, it's lack of documentation.

> My short-mid term goal is to automate as much of the manual process as possible to help eliminate these kinds of mistakes in the future. That way we can focus on the content and not the process.

As you will see, it's already heavily automated. If you want to move the process to something that is clearer for the mobile's team (e.g. no bash+python), that's perfectly fine for me. I don't care on how the results are obtained, I only know what we want:
* Separate per locale commits, so you can actually browse the PR and each locale's diff without looking at all other languages.
* File elements inside XLIFF have target-language set properly for each locale (requires mapping).
* The en-US file contains *translations* for the `en` language.
* Templates is a clean file (without translations and target-languages attributes.

**Pull request closed without merging**